apps/prairielearn/src/pages/instructorAssessmentUploads/instructorAssessmentUploads.html.ts

```diff
@@ -299,3 +326,38 @@ function UploadAssessmentInstanceScoresModal({ csrfToken }: { csrfToken: string
     `,
   });
 }
+
+function UploadSubmissionsCsvModal({ csrfToken }: { csrfToken: string }) {
+  return Modal({
+    id: 'upload-submissions-csv-form',
+    title: 'Upload Submissions CSV',
+    formEncType: 'multipart/form-data',
+    body: html`
+      <p>
+        Upload a CSV file to recreate users, assessment instances, questions, variants, and
+        submissions.
+      </p>
+      <p>You should upload one of the submissions CSV files from the Downloads page.</p>
```

Comment:
Is it worth clarifying here that this does not include the submissions for manual grading file?

Reply:
Sure, why not.

Reply:
Done in 9e90164.